jsplot/src/test/java/tech/tablesaw/plotly/OchlTest.java

```diff
@@ -0,0 +1,47 @@
+package tech.tablesaw.plotly;
+
+import org.junit.jupiter.api.Test;
+import tech.tablesaw.api.DateTimeColumn;
+import tech.tablesaw.api.DoubleColumn;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.plotly.api.OHLCPlot;
+import tech.tablesaw.plotly.components.Figure;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class OchlTest {
```

Comment:
this uses `OHLCPlot`, so it looks like the order of the letters got jumbled in the test name
```suggestion
public class OhlcTest {
```